apis/Google.Cloud.Diagnostics.AspNetCore3/Google.Cloud.Diagnostics.AspNetCore3/Google.Cloud.Diagnostics.AspNetCore3.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All these tabs should be spaces. I'd expect the project generator to rewrite it anyway though.